### PR TITLE
Update to 3.0.0!

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dhirschfeld** [comment]
Just thought I'd try this out...

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
```
RuntimeError: This PR needs a rerender to switch from appveyor to azure
Command exited with code 1
```

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
@conda-forge/core - it looks like this recipe can't be rerendered?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,8 +13,8 @@ source:
   - folder: {{ pkg_src }}
     git_url: https://github.com/helm/helm  # [unix]
     git_tag: v{{ version }}                # [unix]
-    url: https://storage.googleapis.com/kubernetes-{{ name }}/{{ name }}-v{{ version }}-windows-amd64.tar.gz  # [win]
-    sha256: 604780d3fabeb27e7ab7a30c6e29ce64bcd2203501ea35e5231c97965b0255a0  # [win]
+    url: https://get.helm.sh/{{ name }}-v{{ version }}-windows-amd64.zip  # [win]
```
Comment (dhirschfeld) [inline recipe/meta.yaml]: Switched to the url in the GitHub release page as 3.0.0 isn't yet available from `storage.googleapis.com`


https://storage.googleapis.com/kubernetes-helm

File: recipe/meta.yaml [hunk 2]
```diff
@@ -36,7 +36,6 @@ requirements:
 test:
   commands:
     - helm --help
-    - tiller -version
```
Comment (dhirschfeld) [inline recipe/meta.yaml]: tiller begone!

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
Linux error:
```
~/feedstock_root/build_artifacts/kubernetes-helm_1574026195554/work/src/k8s.io/helm ~/feedstock_root/build_artifacts/kubernetes-helm_1574026195554/work
make: *** No rule to make target `bootstrap'.  Stop.
```

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

**Comment by dhirschfeld** [comment]
Thanks @mariusvniekerk for picking this up!

**Pull request merged**